modules/apache-airflow/templates/values.yaml

```diff
@@ -636,7 +638,9 @@ workers:
   extraVolumeMounts: []
 
   # Select certain nodes for airflow worker pods.
-  nodeSelector: {}
+  nodeSelector: {
+    spotinst.io/node-lifecycle: "od"
+  }
```

Comment:
This node selector defined how the spot io cluster autoscaler requests EC2 instances to spin up for the cluster. This makes sure that an on-demand instance is created and that the pods are running on it.